bgbb_airflow/pred_airflow_job.py

```diff
@@ -1,28 +1,48 @@
-from typing import List
+from os.path import join
+from typing import List, Tuple, Union
 
 import click
-from pyspark.sql import SparkSession
-
 from bgbb import BGBB
 from bgbb.sql.bgbb_udfs import mk_n_returns_udf, mk_p_alive_udf
 from bgbb.sql.sql_utils import run_rec_freq_spk
+from pyspark.sql import SparkSession
 
 from bgbb_airflow.bgbb_utils import PythonLiteralOption
 
+default_bucket = "s3://net-mozaws-prod-us-west-2-pipeline-analysis"
+default_prefix = "wbeard/bgbb_preds"
+
 
-def extract(spark, ho_start, model_win=90, sample_ids: List[int] = []):
+def pull_most_recent_params(
+    spark, pars_loc=join(default_bucket, "wbeard/bgbb_params")
+):
```

Comment:
Almost, but this isn't being modified by the click function. This job would probably get run in the following way:

```
python -m bgbb_airflow.pred_airflow_job.main \
    --bucket <BUCKET> \
    --prefix wbeard/bgbb_preds \
    --param-bucket <BUCKET (or some default)> \
    --param-prefix wbeard/bgbb_params
```

The variables can then be modified from within airflow. This isn't necessarily a blocker though if the job will be using the default bucket, although it should be modified at some point.